contrib/cirrus/setup_environment.sh

```diff
@@ -26,70 +35,56 @@ do
     fi
 done
 
-# Setup env. vars common to all tasks/scripts/platforms and
-# ensure they return for every following script execution.
-MARK="# Added by $0, manual changes will be lost."
-touch "$HOME/$ENVLIB"
-if ! grep -q "$MARK" "$HOME/$ENVLIB"
-then
-    cp "$HOME/$ENVLIB" "$HOME/${ENVLIB}_original"
-    # N/B: Single-quote items evaluated every time, double-quotes only once (right now).
-    for envstr in \
-        "$MARK" \
-        "export EPOCH_TEST_COMMIT=\"$CIRRUS_BASE_SHA\"" \
-        "export HEAD=\"$CIRRUS_CHANGE_IN_REPO\"" \
-        "export TRAVIS=\"1\"" \
-        "export GOSRC=\"$CIRRUS_WORKING_DIR\"" \
-        "export OS_RELEASE_ID=\"$(os_release_id)\"" \
-        "export OS_RELEASE_VER=\"$(os_release_ver)\"" \
-        "export OS_REL_VER=\"$(os_release_id)-$(os_release_ver)\"" \
-        "export BUILT_IMAGE_SUFFIX=\"-$CIRRUS_REPO_NAME-${CIRRUS_CHANGE_IN_REPO:0:8}\"" \
-        "export GOPATH=\"/var/tmp/go\"" \
-        'export PATH="$HOME/bin:$GOPATH/bin:/usr/local/bin:$PATH"' \
-        'export LD_LIBRARY_PATH="/usr/local/lib${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}"'
-    do
-        # Make permanent in later shells, and set in current shell
-        X=$(echo "$envstr" | tee -a "$HOME/$ENVLIB") && eval "$X" && echo "$X"
-    done
-
-    # Some setup needs to vary between distros
-    case "${OS_RELEASE_ID}-${OS_RELEASE_VER}" in
-        ubuntu-18)
-            # Always install runc on Ubuntu
-            install_runc_from_git
-            ;;
-        fedora-29)
-            CON_SEL="https://kojipkgs.fedoraproject.org/packages/container-selinux/2.100/1.git3b78187.fc29/noarch/container-selinux-2.100-1.git3b78187.fc29.noarch.rpm"
-            echo ">>>>> OVERRIDING container-selinux WITH $CON_SEL <<<<<"
-            dnf -y install $CON_SEL
-            ;&  # Continue to the next item
-        fedora-28)
-            echo ">>>>> OVERRIDING source-built runc with latest package <<<<<"
-            dnf update -y runc
-            ;&  # Continue to the next item
-        centos-7) ;&
-        rhel-7)
-            ;;
-        *) bad_os_id_ver ;;
-    esac
+# Sometimes environment setup needs to vary between distros
+# Note: This should only be used for environment variables, and temporary workarounds.
+#       Anything externally dependent, should be made fixed-in-time by adding to
+#       contrib/cirrus/packer/*_setup.sh to be incorporated into VM cache-images
+#       (see docs).
+case "${OS_REL_VER}" in
```

Comment:
These can never trigger: OS_REL_VER is <os>-<version>. I think you meant to use OS_RELEASE_ID here.

Reply:
Ya, got it, and found problem in `OS_RELEASE_VER`, is suppose to **only** grab the major version number (e.g. `Ubuntu-18` vs `Ubuntu-18.04`).  Both should be fixed now.